fhir-examples/src/main/resources/json/ibm/basic/BasicComposite.json

```diff
@@ -0,0 +1,144 @@
+{
+  "resourceType": "Basic",
+  "code": {
+    "text": "test"
+  },
+  "extension": [
+    {
+      "url": "http://example.org/composite",
+      "extension": [
+        {
+          "url": "http://example.org/boolean",
+          "valueBoolean": true
+        },
+        {
+          "url": "http://example.org/integer",
+          "valueInteger": 12
+        },
+        {
+          "url": "http://example.org/string",
+          "valueString": "testString"
+        },
+        {
+          "url": "http://example.org/code",
+          "valueCode": "code"
+        },
+        {
+          "url": "http://example.org/id",
+          "valueId": "testId"
+        },
+        {
+          "url": "http://example.org/markdown",
+          "valueMarkdown": "testMarkdown"
+        },
+        {
+          "url": "http://example.org/decimal",
+          "valueDecimal": 99.99
+        },
+        {
+          "url": "http://example.org/uri",
+          "valueUri": "urn:uuid:53fefa32-1111-2222-3333-55ee120877b7"
+        },
+        {
+          "url": "http://example.org/instant",
+          "valueInstant": "2018-10-29T17:12:44-04:00"
+        },
+        {
+          "url": "http://example.org/date",
+          "valueDate": "2018-10-29"
+        },
+        {
+          "url": "http://example.org/dateTime",
+          "valueDateTime": "2018-10-29T17:12:00-04:00"
+        },
+        {
+          "url": "http://example.org/time",
+          "valueTime": "17:12:00"
+        },
+        
```

Comment:
```suggestion
```
Trying to remove the extra line

Reply:
if JSON had comments, this would be a comment to indicate that the "primitive" types are over and now the "complex" types are starting...certainly not obvious from an empty line but still useful to me